src/services/mentees.js

```diff
@@ -365,6 +377,19 @@ module.exports = class MenteesHelper {
 				count: 0,
 			}
 		}
+
+		const defaultRuleFilter = await defaultRulesFilter({
+			ruleType: 'session',
+			requesterId: userId,
+			roles: roles,
+			requesterOrganizationId: orgId,
+		})
+		if (defaultRuleFilter === null) {
```

Comment:
@nevil-mathew if invalid configs you were returning  ' ', please check this.

Reply:
Sending back an empty string so that it won't cause issues in the RAW query